### PR TITLE
add hreflang tags to the weekly page

### PR DESCRIPTION
**Pull request by johnduffell**
This will help google to choose the right page depending where the user is

See https://support.google.com/webmasters/answer/6059209

I also added canonical links to the pages, pointing to the backend generated url for the page.
https://support.google.com/webmasters/answer/139066?hl=en

I have updated digipack in the same way as it came through in the compiler errors when I was doing it.
<details>
  <summary>click to expand</summary>

```html
<!DOCTYPE html>
<html lang="en-GB">
<head>
    


<meta charset="utf-8">

<title>The Guardian Weekly Subscriptions | The Guardian</title>

<meta name="google-site-verification" content="qf7V0ceP_mY_0jTl7R7C1wZSKn2gK7TlharWVLr8Ea0" />
<meta name="google-site-verification" content="usCUaIJGNg9ijq-htmMAU6yEhAwNm7wdyb_fk_s2LTQ" />
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="format-detection" content="telephone=no">
<meta name="HandheldFriendly" content="true">
<meta property="og:title" content="The Guardian Weekly Subscriptions | The Guardian"/>

<meta property="og:type" content="website"/>
    
    <link rel="canonical" href="https://sub.thegulocal.com/weekly/GB">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AF" hreflang="en-AF">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AL" hreflang="en-AL">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/DZ" hreflang="en-DZ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AS" hreflang="en-AS">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AD" hreflang="en-AD">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AO" hreflang="en-AO">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AI" hreflang="en-AI">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AQ" hreflang="en-AQ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AG" hreflang="en-AG">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AR" hreflang="en-AR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AM" hreflang="en-AM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AW" hreflang="en-AW">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AU" hreflang="en-AU">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AT" hreflang="en-AT">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AZ" hreflang="en-AZ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BS" hreflang="en-BS">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BH" hreflang="en-BH">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BD" hreflang="en-BD">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BB" hreflang="en-BB">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BY" hreflang="en-BY">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BE" hreflang="en-BE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BZ" hreflang="en-BZ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BJ" hreflang="en-BJ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BM" hreflang="en-BM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BT" hreflang="en-BT">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BO" hreflang="en-BO">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BQ" hreflang="en-BQ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BA" hreflang="en-BA">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BW" hreflang="en-BW">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BV" hreflang="en-BV">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BR" hreflang="en-BR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/IO" hreflang="en-IO">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/VG" hreflang="en-VG">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BN" hreflang="en-BN">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BG" hreflang="en-BG">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BF" hreflang="en-BF">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BI" hreflang="en-BI">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/KH" hreflang="en-KH">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CM" hreflang="en-CM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CA" hreflang="en-CA">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CV" hreflang="en-CV">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/KY" hreflang="en-KY">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CF" hreflang="en-CF">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TD" hreflang="en-TD">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CL" hreflang="en-CL">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CN" hreflang="en-CN">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CX" hreflang="en-CX">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CC" hreflang="en-CC">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CO" hreflang="en-CO">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/KM" hreflang="en-KM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CG" hreflang="en-CG">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CD" hreflang="en-CD">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CK" hreflang="en-CK">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CR" hreflang="en-CR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/HR" hreflang="en-HR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CU" hreflang="en-CU">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CW" hreflang="en-CW">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CY" hreflang="en-CY">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CZ" hreflang="en-CZ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/DK" hreflang="en-DK">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/DJ" hreflang="en-DJ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/DM" hreflang="en-DM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/DO" hreflang="en-DO">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TL" hreflang="en-TL">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/EC" hreflang="en-EC">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/EG" hreflang="en-EG">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SV" hreflang="en-SV">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GQ" hreflang="en-GQ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/ER" hreflang="en-ER">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/EE" hreflang="en-EE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/ET" hreflang="en-ET">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/FO" hreflang="en-FO">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/FK" hreflang="en-FK">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/FJ" hreflang="en-FJ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/FI" hreflang="en-FI">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/FR" hreflang="en-FR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GF" hreflang="en-GF">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PF" hreflang="en-PF">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TF" hreflang="en-TF">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GA" hreflang="en-GA">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GM" hreflang="en-GM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GE" hreflang="en-GE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/DE" hreflang="en-DE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GH" hreflang="en-GH">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GI" hreflang="en-GI">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GR" hreflang="en-GR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GL" hreflang="en-GL">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GD" hreflang="en-GD">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GP" hreflang="en-GP">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GU" hreflang="en-GU">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GT" hreflang="en-GT">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GG" hreflang="en-GG">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GN" hreflang="en-GN">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GW" hreflang="en-GW">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GY" hreflang="en-GY">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/HT" hreflang="en-HT">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/HM" hreflang="en-HM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/VA" hreflang="en-VA">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/HN" hreflang="en-HN">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/HK" hreflang="en-HK">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/HU" hreflang="en-HU">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/IS" hreflang="en-IS">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/IN" hreflang="en-IN">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/ID" hreflang="en-ID">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/IR" hreflang="en-IR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/IQ" hreflang="en-IQ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/IE" hreflang="en-IE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/IM" hreflang="en-IM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/IL" hreflang="en-IL">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/IT" hreflang="en-IT">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CI" hreflang="en-CI">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/JM" hreflang="en-JM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/JP" hreflang="en-JP">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/JE" hreflang="en-JE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/JO" hreflang="en-JO">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/KZ" hreflang="en-KZ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/KE" hreflang="en-KE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/KI" hreflang="en-KI">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/KW" hreflang="en-KW">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/KG" hreflang="en-KG">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/LA" hreflang="en-LA">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/LV" hreflang="en-LV">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/LB" hreflang="en-LB">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/LS" hreflang="en-LS">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/LR" hreflang="en-LR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/LY" hreflang="en-LY">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/LI" hreflang="en-LI">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/LT" hreflang="en-LT">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/LU" hreflang="en-LU">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MO" hreflang="en-MO">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MK" hreflang="en-MK">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MG" hreflang="en-MG">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MW" hreflang="en-MW">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MY" hreflang="en-MY">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MV" hreflang="en-MV">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/ML" hreflang="en-ML">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MT" hreflang="en-MT">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MH" hreflang="en-MH">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MQ" hreflang="en-MQ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MR" hreflang="en-MR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MU" hreflang="en-MU">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/YT" hreflang="en-YT">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MX" hreflang="en-MX">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/FM" hreflang="en-FM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MD" hreflang="en-MD">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MC" hreflang="en-MC">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MN" hreflang="en-MN">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/ME" hreflang="en-ME">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MS" hreflang="en-MS">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MA" hreflang="en-MA">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MZ" hreflang="en-MZ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MM" hreflang="en-MM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/NA" hreflang="en-NA">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/NR" hreflang="en-NR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/NP" hreflang="en-NP">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/NL" hreflang="en-NL">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/NC" hreflang="en-NC">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/NZ" hreflang="en-NZ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/NI" hreflang="en-NI">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/NE" hreflang="en-NE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/NG" hreflang="en-NG">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/NU" hreflang="en-NU">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/NF" hreflang="en-NF">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/KP" hreflang="en-KP">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MP" hreflang="en-MP">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/NO" hreflang="en-NO">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/OM" hreflang="en-OM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PK" hreflang="en-PK">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PW" hreflang="en-PW">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PS" hreflang="en-PS">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PA" hreflang="en-PA">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PG" hreflang="en-PG">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PY" hreflang="en-PY">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PE" hreflang="en-PE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PH" hreflang="en-PH">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PN" hreflang="en-PN">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PL" hreflang="en-PL">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PT" hreflang="en-PT">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PR" hreflang="en-PR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/QA" hreflang="en-QA">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/RO" hreflang="en-RO">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/RU" hreflang="en-RU">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/RW" hreflang="en-RW">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/RE" hreflang="en-RE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/BL" hreflang="en-BL">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/KN" hreflang="en-KN">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SH" hreflang="en-SH">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/LC" hreflang="en-LC">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/MF" hreflang="en-MF">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/PM" hreflang="en-PM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/VC" hreflang="en-VC">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/WS" hreflang="en-WS">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SM" hreflang="en-SM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/ST" hreflang="en-ST">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SA" hreflang="en-SA">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SN" hreflang="en-SN">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/RS" hreflang="en-RS">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SC" hreflang="en-SC">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SL" hreflang="en-SL">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SG" hreflang="en-SG">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SX" hreflang="en-SX">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SK" hreflang="en-SK">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SI" hreflang="en-SI">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SB" hreflang="en-SB">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SO" hreflang="en-SO">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/ZA" hreflang="en-ZA">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GS" hreflang="en-GS">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/KR" hreflang="en-KR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/ES" hreflang="en-ES">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/LK" hreflang="en-LK">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SD" hreflang="en-SD">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SR" hreflang="en-SR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SJ" hreflang="en-SJ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SZ" hreflang="en-SZ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SE" hreflang="en-SE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/CH" hreflang="en-CH">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/SY" hreflang="en-SY">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TW" hreflang="en-TW">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TJ" hreflang="en-TJ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TZ" hreflang="en-TZ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TH" hreflang="en-TH">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TG" hreflang="en-TG">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TK" hreflang="en-TK">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TO" hreflang="en-TO">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TT" hreflang="en-TT">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TN" hreflang="en-TN">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TR" hreflang="en-TR">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TM" hreflang="en-TM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TC" hreflang="en-TC">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/TV" hreflang="en-TV">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/UG" hreflang="en-UG">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/UA" hreflang="en-UA">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AE" hreflang="en-AE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/GB" hreflang="en-GB">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/US" hreflang="en-US">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/UM" hreflang="en-UM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/VI" hreflang="en-VI">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/UY" hreflang="en-UY">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/UZ" hreflang="en-UZ">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/VU" hreflang="en-VU">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/VE" hreflang="en-VE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/VN" hreflang="en-VN">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/WF" hreflang="en-WF">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/EH" hreflang="en-EH">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/YE" hreflang="en-YE">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/ZM" hreflang="en-ZM">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/ZW" hreflang="en-ZW">
        
        <link rel="alternate" href="https://sub.thegulocal.com/weekly/AX" hreflang="en-AX">
        
    




<script src="https://polyfill.guim.co.uk/v2/polyfill.min.js?features=default-3.6,Element.prototype.dataset,Array.prototype.includes"></script>


```

</details>

@paulbrown1982 @pvighi

### CODE REVIEW

File: app/controllers/WeeklyLandingPage.scala [hunk 1]
```diff
@@ -16,17 +17,28 @@ object WeeklyLandingPage extends Controller {
...
+    val parsedCountry = if (country == international) Some(Country.UK) else CountryGroup.countryByCode(country)
+    parsedCountry.fold {
       MovedPermanently(routes.WeeklyLandingPage.withCountry(international).url)
-    } {country =>
-      Ok(weeklyLandingPage(country, catalog, None, None,description, PegdownMarkdownRenderer))
+    } { country =>
+      val hreflangs = CountryGroup.countries.map { country =>
+        Hreflang(Config.subscriptionsUrl + routes.WeeklyLandingPage.withCountry(country.alpha2).url, s"en-${country.alpha2}")
+      }
+      val hreflang = Hreflangs(Config.subscriptionsUrl + routes.WeeklyLandingPage.withCountry("int").url, hreflangs)
```
Comment (paulbrown1982) [inline app/controllers/WeeklyLandingPage.scala]: Maybe we can rewrite the /weekly/int version to just be /weekly?

e.g.

```
def index(country: Option[Country]) = NoCacheAction { implicit request =>
    val maybeCountry = country orElse request.getFastlyCountry
    maybeCountry.map { c => 
          Redirect(routes.WeeklyLandingPage.withCountry(c))		  
    } getOrElse {
          Ok(routes.WeeklyLandingPage.withCountry(international))
    }
}
```

**Comment by johnduffell** [comment]
I'll take out canonical as it's not adding anything

File: app/controllers/PromoLandingPage.scala [hunk 2]
```diff
@@ -81,14 +80,14 @@ object PromoLandingPage extends Controller {
     } else if ((applicableRatePlanIds intersect paperOnlyPackageRatePlanIds).nonEmpty) {
       Some(routes.Shipping.viewCollectionPaper())
     } else if ((applicableRatePlanIds intersect guardianWeeklyRatePlanIds).nonEmpty) {
-      Some(routes.WeeklyLandingPage.index(maybeCountry))
+      Some(routes.WeeklyLandingPage.index(Some(country)))
```
Comment (paulbrown1982) [inline app/controllers/PromoLandingPage.scala]: If we're not going to delegate country lookup to WeeklyLandingPage then you can use WeeklyLandingPage.withCountry instead.

**Comment by johnduffell** [comment]
actually I added canonical back in, because that means we don't have to worry about campaign codes etc messing with the seo

**Comment by johnduffell** [comment]
hard to test... I've submitted an index request to google but I can't tell if it is working!

**Comment by johnduffell** [comment]
this has worked! 6200 tags on the site according to webmaster tools
image

**Pull request merged**